docs/_includes/date-picker.html

```diff
@@ -1 +1,282 @@
+                    </td>
+                </tr>
+                <tr>
+                    <td class="calendar-month__day calendar-month__day--range">
+                        <button type="button" aria-label="Sunday 25 August 2019">25</button>
+                    </td>
+                    <td class="calendar-month__day calendar-month__day--range">
+                        <button type="button" aria-label="Monday 26 August 2019">26</button>
+                    </td>
+                    <td class="calendar-month__day calendar-month__day--range calendar-month__day--end">
+                        <button type="button" aria-label="Tuesday 27 August 2019" aria-selected="true">27</button>
+                    </td>
+                    <td class="calendar-month__day">
+                        <button type="button" aria-label="Wednesday 28 August 2019">28</button>
+                    </td>
+                    <td class="calendar-month__day">
+                        <button type="button" aria-label="Thursday 29 August 2019">29</button>
+                    </td>
+                    <td class="calendar-month__day">
+                        <button type="button" aria-label="Friday 30 August 2019">30</button>
+                    </td>
+                    <td class="calendar-month__day">
+                        <button type="button" aria-label="Saturday 31 August 2019">31</button>
+                    </td>
+                </tr>
+            </table>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="date-picker">
+    <div class="calendar-month">
+        <div class="calendar-month__header">
+            <button class="date-picker__month-button date-picker__month-button--left icon-btn icon-btn--transparent" aria-label="Go to July 2019" variant="icon">
+                <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
+                    <use href="#icon-chevron-left"></use>
+                </svg>
+            </button>
+            <h3>August 2019</h3>
+            <button class="date-picker__month-button date-picker__month-button--right icon-btn icon-btn--transparent" aria-label="Go to July 2019" variant="icon">
+                <svg aria-hidden="true" class="icon icon--chevron-right" focusable="false">
+                    <use href="#icon-chevron-right"></use>
+                </svg>
+            </button>
+        </div>
+        <table class="calendar-month__body" aria-label="August 2019">
+            <tr>
+                <th>Su</th>
+                <th>Mo</th>
+                <th>Tu</th>
+                <th>We</th>
+                <th>Th</th>
+                <th>Fr</th>
+                <th>Sa</th>
+            </tr>
+            <tr>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td class="calendar-month__day">
+                    <button type="button" aria-label="Thursday 1 August 2019" disabled>1</button>
```

Comment:
I included these labels because the old Marko date picker does the same. If there is a better way I am happy to change the format or remove them. What are your thoughts @ianmcburnie?

Reply:
For voice input technology, the start of the aria-label should match the visible label. So if we do add aria-labels, it would be `aria-label="1 - Thursday, August 2019". For now though, let's leave them off. The context can be derived from the table semantics.